roles/marathon/tasks/main.yml

```diff
@@ -9,7 +8,7 @@
     - marathon
 
 - name: create marathon artifact store directory
-  when: marathon_artifact_store_dir is defined and marathon_enabled | bool
+  when: marathon_artifact_store_dir is defined
```

Comment:
do this marathon_artifact_store_dir is defined and marathon_artifact_store_dir != ''